internal/services/oracle/oracle_autonomous_database_backup_data_source.go

```diff
@@ -0,0 +1,221 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package oracle
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/oracledatabase/2025-03-01/autonomousdatabasebackups"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/oracledatabase/2025-03-01/autonomousdatabases"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+)
+
+type AutonomousDatabaseBackupDataSource struct{}
+
+type AutonomousDatabaseBackupsDataModel struct {
+	AutonomousDatabaseId      string                              `tfschema:"autonomous_database_id"`
+	AutonomousDatabaseBackups []AutonomousDatabaseBackupDataModel `tfschema:"autonomous_database_backups"`
+}
+type AutonomousDatabaseBackupDataModel struct {
+	Id                           string  `tfschema:"id"`
+	DisplayName                  string  `tfschema:"display_name"`
+	BackupType                   string  `tfschema:"backup_type"`
+	RetentionPeriodInDays        int64   `tfschema:"retention_period_in_days"`
+	AutonomousDatabaseOcid       string  `tfschema:"autonomous_database_ocid"`
+	AutonomousDatabaseBackupOcid string  `tfschema:"autonomous_database_backup_ocid"`
+	DbVersion                    string  `tfschema:"database_version"`
+	BackupSizeInTbs              float64 `tfschema:"database_backup_size_in_tbs"`
+	IsAutomatic                  bool    `tfschema:"is_automatic"`
+	IsRestorable                 bool    `tfschema:"is_restorable"`
```

Comment:
Could we shorten these?
```suggestion
	Automatic                  bool    `tfschema:"automatic"`
	Restorable                 bool    `tfschema:"restorable"`
```